index.bs

```diff
@@ -9915,6 +9911,12 @@ Constructors</h5>
 			1. Let <var>node</var> be the instance being created by the
```

Comment:
Pre-existing, but you should just be able to reference `this` here, per https://heycam.github.io/webidl/#create-an-interface-object step 1.9

Reply:
Do you propose to replace `node` with `this`? or do you mean I can just use `this` and remove this step?

Reply:
You can just say "Let _node_ be the **this** value" with a link to https://heycam.github.io/webidl/#this if desired.

Reply:
Good to know. I'll address this in a different PR.